### PR TITLE
Task/shank 247 ingress support stu3

### PR DESCRIPTION
**Pull request by jhulbert-va**
[SHANK-247](https://vasdvp.atlassian.net/browse/SHANK-247)

This PR should address the issue with getting an stu3 Location response from data-query.

@beaugrantham-va @jblefkove-va 
I tested the following apps (below) with these changes. Let me know if I missed an app that uses the `/fhr/v0/` path and I will make sure the ingress change does not affect it.

- data-query
- provider-directory
- urgent-care
- claims

### CODE REVIEW

File: products/data-query.yaml [hunk 1]
```diff
@@ -24,61 +24,88 @@ metadata:
...
   annotations:
     kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/rewrite-target: /dstu2/$1
+    nginx.ingress.kubernetes.io/rewrite-target: /$1/$2$3
 spec:
   rules:
     - http:
         paths:
-          - path: /fhir/v0/dstu2/(.*)
+          - path: /fhir/v0/(dstu2|stu3|r4)/(AllergyIntolerance|Appointment|Condition|DiagnosticReport|Encounter|Immunization|Location|Medication|MedicationDispense|MedicationOrder|MedicationStatement|Observation|Organization|Patient|Practitioner|Procedure)(.*)
```
Comment (jblefkove-va) [inline products/data-query.yaml]: Do you want to add `PractitionerRole` here as well? (That is not in data-query yet, but will be soon.)

File: products/data-query.yaml [hunk 2]
```diff
@@ -24,61 +24,88 @@ metadata:
...
   annotations:
     kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/rewrite-target: /dstu2/$1
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
 spec:
   rules:
     - http:
         paths:
-          - path: /fhir/v0/argonaut/data-query/(.*)
+          - path: /fhir/v0/(dstu2/metadata)
```
Comment (beaugrantham-va) [inline products/data-query.yaml]: Who is going to host stu3/metadata? PD or DQ?

Comment (bschofield-va) [reply products/data-query.yaml]: ultimately the unifier, currently DQ doesn't have stu3 metadata

Comment (beaugrantham-va) [reply products/data-query.yaml]: I guess where I was going with this was that the unifier needs to know where to scrape that endpoint. We'll assume that PD continues to be that source for now.

File: products/data-query.yaml (hunk 1 above)
Comment (jhulbert-va) [reply products/data-query.yaml]: I think adding practitionerRole would be done as part of the process when adding it to DQ I don't think we should add it into the ingress until thats done.

**Pull request merged**